osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -249,11 +262,13 @@ def execute_task(extraction_order_id, supported_export_formats, execution_config
                     },
                     email=False
                 )
+                logger.exception()
```

Comment:
Python docs state:
```
Logger.exception(msg, *args, **kwargs)
Logs a message with level ERROR on this logger. The arguments are interpreted as for debug(). Exception info is added to the logging message. This method should only be called from an exception handler.
```

Doesn't that mean, I don't have to add any information, because it will be added to the log-message anyway?